website/docs/script/sudt-script.mdx

```diff
@@ -0,0 +1,383 @@
+    let out_point = context.deploy_cell(contract_bin);
+    let type_script = context
+        .build_script(&out_point, Bytes::from(vec![42]))
+        .expect("script");
+    let type_script_dep = CellDep::new_builder().out_point(out_point).build();
+
+    let input_token: u128 = 400;
+    let output_token1: u128 = 300;
+    let output_token2: u128 = 100;
+
+    // prepare cells
+    let input_out_point = context.create_cell(
+        CellOutput::new_builder()
+            .capacity(1000u64.pack())
+            .lock(lock_script.clone())
+            .type_(Some(type_script.clone()).pack())
+            .build(),
+        input_token.to_le_bytes().to_vec().into(),
+    );
+    let input = CellInput::new_builder()
+        .previous_output(input_out_point)
+        .build();
+    let outputs = vec![
+        CellOutput::new_builder()
+            .capacity(500u64.pack())
+            .lock(lock_script.clone())
+            .type_(Some(type_script.clone()).pack())
+            .build(),
+        CellOutput::new_builder()
+            .capacity(500u64.pack())
+            .lock(lock_script)
+            .type_(Some(type_script).pack())
+            .build(),
+    ];
+
+    let outputs_data = vec![
+        Bytes::from(output_token1.to_le_bytes().to_vec()),
+        Bytes::from(output_token2.to_le_bytes().to_vec()),
+    ];
+
+    // build transaction
+    let tx = TransactionBuilder::default()
+        .input(input)
+        .outputs(outputs)
+        .outputs_data(outputs_data.pack())
+        .cell_dep(lock_script_dep)
+        .cell_dep(type_script_dep)
+        .build();
+    let tx = context.complete_tx(tx);
+
+    // run
+    let cycles = context
+        .verify_tx(&tx, 10_000_000)
+        .expect("pass verification");
+    println!("consume cycles: {}", cycles);
+}
+```
+
+In this test case:
+
+- We deploy the sudt Script we wrote and use it to build cell's type script.
+- We set up the initial conditions by defining the valid input and output amounts.
+- Finally, We build the full transaction and ensure that the transaction can be verified.
+
+Notice that in the type script args we use a random bytes which is 42 indicating that we are not under the `owner_mode` in the `test_transfer_sudt` test case.
+
+For ownermode testcase, you can checkout the [full tests code](https://github.com/nervosnetwork/docs.nervos.org/tree/develop/examples/sudt-script/contracts/sudt/tests/src/tests.rs) for reference.
+
+By writing tests like `test_transfer_sudt`, we can ensure that our sudt script behaves correctly under various scenarios, helping to catch any issues before deployment.
+
+---
+
+## Congratulations!
+
+By following this tutorial so far, you have mastered how to write a simple UDT Script that brings fungible tokens to CKB.
+Here's a quick recap:
+
+- Use `offckb` and `ckb-script-templates` to init a Script project
+- Use `ckb_std` to leverage CKB syscalls `Source::GroupIuput`/`Source::GroupOutput` for performing relevant cells iteration.
```

Comment:
- Use `ckb_std` to leverage CKB syscalls `Source::GroupIuput`/`Source::GroupOutput` for performing relevant Cell iteration.